extensions/multirecordings.sigmf-ext.md

```diff
@@ -0,0 +1,128 @@
+# The `multirecordings` SigMF Extension Namespace v0.1.0
+
+This extension makes it possible to link multiple SigMF recordings together to
+represent a single data-capture event. This is primarily used for two purposes:
+representing dynamic aspects of one SigMF recording and
+capturing multi-channel data.
+
+For recordings that digitize multiple streams of information (e.g., a
+phased-array receiver with multiple channels), this extension enables you to
+capture the multiple streams and link them such that each one can be treated as
+a separate `SigMF Recording` or they may be treated together as one.
+
+For some recordings, there are fields that are continuously variable. Examples
+include the geolocation of a radio receiver that is on a moving vehicle, the
+azimuth of a spinning antenna aperture, and the temperature readings of a
+thermal sensor. If these were stored as metadata in the "primary" signal
+recording, they would bloat and complicate the use of that recording.
+Furthermore, it would be difficult to process any aspect of the data without
+parsing all of the recorded data. This extension enables you to treat such
+fields as *simply another `SigMF Recording`*.
+
+Per the SigMF specification, all recordings must be independently evaluated and
+be able to stand-alone as compliant recordings.
+
+## 0 Datatypes
+
+| type      | long-form name        | description                                             |
+| --------- | --------------------- | ------------------------------------------------------- |
+| recording | string recording name | A string that indicates the name of a `SigMF Recording` |
+
+The `recording` datatype is the primary mechanism by which SigMF Recordings are
+linked. This datatype is the string name of a SigMF Recording, **without** filename extensions
+(i.e., given a recording comprising `N.sigmf-meta` and `N.sigmf-data`, the
+`recording` datatype value would simply be `N`).
+
+This datatype may take the place of any non-string datatype in a metadata file. The presence of string filename then indicates that particular field in the recording is described by another recording with the base name in the string.
+
+The second recording (both metadata and dataset files) MUST be in the same directory
+(i.e., local to the SigMF Recording in which they are referenced, and thus in
+the same SigMF Archive).
+
+## 1 Global
+
+This extension adds the following optional fields to the `Global` object.
+
+| name          | required | type      | description                         |
+| ------------- | -------- | --------- | ----------------------------------- |
+| `primary`     | false    | recording | The primary recording this recording is linked to.|
+| `channel`     | false    | uint      | The channel index of this recording.|
+| `streams`     | false    | object    | List of SigMF recordings that represent multiple streams.|
```

Comment:
Maybe this should be slightly reworded to be more specific:

| `streams`     | false    | object    | List of SigMF `recording` objects that represent multiple streams.|